package.json

```diff
@@ -44,30 +44,30 @@
 		"prepare": "npm run build && npm rebuild --build-from-source gl"
 	},
 	"dependencies": {
-		"@assemblyscript/loader": "^0.19.3",
+		"@as-pect/cli": "^7.0.7",
```

Comment:
Was there a particular reason to move the cli to `dependencies`? Or just accidental?